index.html

```diff
@@ -619,10 +639,14 @@ <h2 class="title is-4">Best airways</h2>
     <button class="modal-close is-large visually-hidden">modal close</button>
   </div>
 
-  <div id="modal-image1" class="modal modal-fx-3dSlit">
+  <div
+    id="modal-image1" 
+    class="modal modal-fx-3dSlit" 
+    role="dialog"
+    aria-labelledby="dialog2Img">
```

Comment:
change aria-labelledby to aria-label. It's about description what modal window is this